containers/ruby/.devcontainer/library-scripts/ruby-debian.sh

```diff
@@ -247,25 +247,42 @@ git clone --depth=1 \
     -c fetch.fsck.zeroPaddedFilemode=ignore \
     -c receive.fsck.zeroPaddedFilemode=ignore \
     https://github.com/rbenv/rbenv.git /usr/local/share/rbenv
-ln -s /usr/local/share/rbenv/bin/rbenv /usr/local/bin
-updaterc 'eval "$(rbenv init -)"'
+
 git clone --depth=1 \
     -c core.eol=lf \
     -c core.autocrlf=false \
     -c fsck.zeroPaddedFilemode=ignore \
     -c fetch.fsck.zeroPaddedFilemode=ignore \
     -c receive.fsck.zeroPaddedFilemode=ignore \
     https://github.com/rbenv/ruby-build.git /usr/local/share/ruby-build
+
 mkdir -p /root/.rbenv/plugins
-ln -s /usr/local/share/ruby-build /root/.rbenv/plugins/ruby-build
+
 if [ "${USERNAME}" != "root" ]; then
     mkdir -p /home/${USERNAME}/.rbenv/plugins
-    chown -R ${USERNAME} /home/${USERNAME}/.rbenv
-    ln -s /usr/local/share/ruby-build /home/${USERNAME}/.rbenv/plugins/ruby-build
 fi
 
+# Update bashrc and zshrc with conditionals
+# We use "not false" so that the default state is true, i.e., enabled
+updaterc 'if [ "${RVM_ENABLED}" != "false" ]; then 
+    source /usr/local/rvm/scripts/rvm 
+fi'
+
+if [ "${USERNAME}" != "root" ]; then
+    mkdir -p /home/${USERNAME}/.rbenv/plugins
+    updaterc 'if [ "${RBENV_ENABLED}" != "false" ]; then
```

Comment:
Not sure why not using: `if [ "${RBEN_ENABLED} = "true" ] ..` . It could be that I can initialize with anything and this statement will be true always. (i.e `export RBEN_ENABLED = "blah"`)

Reply:
We actually want this to default to being enabled, and for users to explicitly disable it. In talking to Chuck, the user base that wants and needs the config files from RBENV and RVM is larger than the base that wants to disable it. So we want to the tools to be default enabled and explicitly disabled, rather than the reverse.

Reply:
hi @KanishkT123 , I think if your above statement is accurate, then the `devcontainer.json` _containerEnv_ vars should have the values set to "true" rather than "false" ... ?

Reply:
Correct, I actually noticed and updated that in the last commit!

Reply:
> We actually want this to default to being enabled, and for users to explicitly disable it. In talking to Chuck, the user base that wants and needs the config files from RBENV and RVM is larger than the base that wants to disable it. So we want to the tools to be default enabled and explicitly disabled, rather than the reverse.

Seems what you want  is to know whether the env variable exists, rather than the value of it. I would rather use `[ -z "${RBENV}" ]` to check this condition and to avoid confusion on the actual value of the variable.